src/Microsoft.Health.Fhir.Core/Configs/CorsConfiguration.cs

```diff
@@ -0,0 +1,17 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Microsoft.Health.Fhir.Core.Features.Cors;
+
+namespace Microsoft.Health.Fhir.Core.Configs
+{
+    public class CorsConfiguration
```

Comment:
I think we may want this to be more customizable (perhaps looking like `CorsPolicy`). Also, I would have guessed that `SupportsCredentials` would be needed for SMART on FHIR?

Reply:
I refactored this to be more configurable.